### ISSUE TITLE
read-pwr-ad7998

### ISSUE THREAD

**Issue by A-Hubbell** [spaceconcordia/space-jobs#7]
Line 41 comment shows the path to "in_voltage5_raw" but the function is calling the device name "in_voltage6_raw".

**Comment by SpaceShawn** [comment]
I'm assuming you're referring to the development branch here? This is a question best answered by Shawn Stoute because I'm not sure which I2C outputs we are interested in. 

Here is the line in question:
```
/**** /sys/bus/i2c/devices/1-0023/iio:device3/in_voltage5_raw : channel 6 - solar panel1_3 voltage divider ****/
readDevice(pPath, "/in_voltage6_raw");
```

The comment refers to in_voltage5_raw, however in_voltage6_raw is actually being read. Which one is correct?

**Comment by A-Hubbell** [comment]
Yes I was referring to the development branch which I'm working out of, but the same issue is in the master branch.

**Comment by SpaceShawn** [comment]
Is this issue resolved?
